tests/Application/src/Tests/Validator/TranslatableValidatorTest.php

```diff
@@ -0,0 +1,67 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Sylius Sp. z o.o.
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace App\Tests\Validator;
+
+use ApiTestCase\JsonApiTestCase;
+use App\Entity\Book;
+use Symfony\Component\Validator\Validator\ValidatorInterface;
+
+final class TranslatableValidatorTest extends JsonApiTestCase
```

Comment:
```suggestion
final class TranslatableValidatorTest extends KernelTestCase
```
🤔

Reply:
I used the `loadFixturesFromFile` method, so I need to extend `JsonApiTestCase`.